ddtrace/llmobs/_llmobs.py

```diff
@@ -382,10 +382,11 @@ def _llmobs_span_event(self, span: Span) -> Dict[str, Any]:
     @staticmethod
     def _llmobs_tags(span: Span) -> List[str]:
         tags = [
-            "version:%s" % (config.version or ""),
-            "env:%s" % (config.env or ""),
-            "service:%s" % (span.service or ""),
+            "version:{}".format(config.version or ""),
```

Comment:
But: We could use f strings here

Reply:
Will do these as a bigger cleanup effort in a future PR